pkg/kubelet/container_gc.go

```diff
@@ -79,6 +86,12 @@ type containerGCInfo struct {
 
 	// Creation time for the container.
 	createTime time.Time
+
+	// Pod name
+	podName string
```

Comment:
From https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/kubelet/dockertools/docker.go#L273 this podName is in 'namespace_podName' format.